SWG-Source/dsrc
# GU 16.9 - Remove Unique Status from Heroic Jewelry

**Summary**: A fairly easy one that just updates the master item table to remove the unique flag on heroic sets.

Addresses: [https://github.com/orgs/SWG-Source/projects/1#card-20558390](https://github.com/orgs/SWG-Source/projects/1#card-20558390)

Original Patch Notes: [https://swg.fandom.com/wiki/Hotfix_16.9](https://swg.fandom.com/wiki/Hotfix_16.9)

## Clarifications Required

There are four jewellery sets within the master items table I don't recognise. As they appear to be Heroic sets, I've grouped them into this change, but it would be good to confirm that this is correct.

- The Tinkerer's Set
- The Rebel Siege GCW Set
- The Imperial Siege GCW Set
- "Testing" set

## **Testing**

1. Go to a Frog, spawn a heroic set with `Armor` → `Heroic Jewelry` → `Bounty Hunter Enforcer (DPS)`
2. Check whether the item has the `unique` status within your inventory.

**Before**
![SwgClient_r_zg71cSCekb](https://user-images.githubusercontent.com/762130/93656313-4a8f6e00-fa21-11ea-9a2d-f397affea6a8.png)

**After**
![SwgClient_r_mlCohHc98Z](https://user-images.githubusercontent.com/762130/93656314-4ebb8b80-fa21-11ea-9c26-b68fcfa2ae03.png)
